pbs_light.go

```diff
@@ -378,26 +378,51 @@ func (deps *auctionDeps) auction(w http.ResponseWriter, r *http.Request, _ httpr
 		}
 	}
 	if pbs_req.CacheMarkup == 1 {
-		cobjs := make([]*pbc.CacheObject, len(pbs_resp.Bids))
-		for i, bid := range pbs_resp.Bids {
-			bc := &pbc.BidCache{
-				Adm:    bid.Adm,
-				NURL:   bid.NURL,
-				Width:  bid.Width,
-				Height: bid.Height,
+		var cobjs []*pbc.CacheObject
+		var cxmls []*pbc.CacheXmlObject
+		for _, bid := range pbs_resp.Bids {
+			if bid.CreativeMediaType == "video" {
+				cxmls = append(cxmls, &pbc.CacheXmlObject{
+					Value: bid.Adm,
+				})
+			} else {
+				bc := &pbc.BidCache{
+					Adm:    bid.Adm,
+					NURL:   bid.NURL,
+					Width:  bid.Width,
+					Height: bid.Height,
+				}
+				cobjs = append(cobjs, &pbc.CacheObject{
+					Value: bc,
+				})
 			}
-			cobjs[i] = &pbc.CacheObject{
-				Value: bc,
+		}
+		if len(cobjs) > 0 {
+			err = pbc.Put(ctx, cobjs)
+			if err != nil {
+				writeAuctionError(w, "Prebid cache failed", err)
+				mErrorMeter.Mark(1)
+				return
 			}
 		}
-		err = pbc.Put(ctx, cobjs)
-		if err != nil {
-			writeAuctionError(w, "Prebid cache failed", err)
-			mErrorMeter.Mark(1)
-			return
+		if len(cxmls) > 0 {
+			err = pbc.PutXml(ctx, cxmls)
+			if err != nil {
+				writeAuctionError(w, "Prebid cache failed", err)
+				mErrorMeter.Mark(1)
+				return
+			}
 		}
-		for i, bid := range pbs_resp.Bids {
-			bid.CacheID = cobjs[i].UUID
+		objIndex := 0
```

Comment:
objIndex, xmlIndex := 0

Reply:
I think if we want to declare variables in one line it should be `objIndex, xmlIndex := 0, 0`, both one line declaration and two line declaration look good to me.